lessons/Part2.Rmd

```diff
@@ -68,19 +68,40 @@ Beyond mathematical operations, we can apply **functions** to vectors, for examp
 length(pop_mexico)
 ```
 
-Functions are commands that take an input and return some output. They are identified by the trailing brackets. Inside the brackets you'll put:
+Functions are commands that take an **input** and return some **output**. They are identified by the trailing brackets. Inside the brackets go inputs (also called **arguments**). Each input takes a value that we provide when calling the function. 
```

Comment:
change bracket to parentheses (there are two places)

i would change these to be a little more clear and casual, something like "You can always tell you're working with a function when you see a pair of parentheses following a function name"

Reply:
Flesh this out to make it a little more clear:

Inside the brackets go inputs -> 
"The inputs to the function go inside the parentheses. You might see these inputs also referred to as **arguments** or **parameters**. Each input has a name, but you might not always see the names of the arguments when a function is being called.

When you call a function, you a provide a value for each argument. The function uses those values to generate its output"

Reply:
and perhaps make it explicit that we will simply call these inputs arguments from here on out, and be strict about only using arguments as the term